_includes/common/ui/statements-page.md

```diff
@@ -23,6 +23,8 @@ To filter by [application]({{ link_prefix }}connection-parameters.html#additiona
 
 You can also filter by database (**Database**), SQL statement type (**Statement Type**), and for [statement fingerprints](#sql-statement-fingerprints) that take longer than a specified time to run. To display only statements with queries that cause full table scans, click **Only show statements that contain queries with full table scans**.
 
+Click <img src="{{ 'images/v21.2/ui-columns-button.png' | relative_url }}" alt="Download bundle" /> to select the columns to display.
```

Comment:
Shouldn't the alt text be something like "Column selector icon"?

Since this glyph is common to both the v21.2 DB Console and the Cloud Console, should we create a new `images/common` directory for these kind of images?

Reply:
Great catch about alt text. Thanks.  Agreed on moving the glyph.